INSTALL

```diff
@@ -933,6 +948,10 @@ The CMake build system adds auto-detection for the following packages/libraries:
  - POSIXCLOCKS
  - PUMI
  - HIOP
+ - ADEPT
+ - FADBAD++
+ - CoDiPack
+ - Eigen
```

Comment:
Is this correct?

Reply:
ADEPT/FADBAD++/EIGEN will be available in the next PR - I have removed from the INSTALL file